deploy/helm/sumologic/values.yaml

```diff
@@ -103,6 +103,28 @@ sumologic:
   ## Verify SumoLogic HTTPS certificates
   verifySsl: "true"
 
+  ## A regular expression for containers. 
+  ## Matching containers will be excluded from Sumo. The logs will still be sent to FluentD.
+  excludeContainerRegex: ""
+
+  ## A regular expression for hosts. 
+  ## Matching hosts will be excluded from Sumo. The logs will still be sent to FluentD.
+  excludeHostRegex: ""
+
+  ## A regular expression for namespaces. 
+  ## Matching namespaces will be excluded from Sumo. The logs will still be sent to FluentD.
+  excludeNamespaceRegex: ""
+
+  ## Files matching this pattern will be ignored by the in_tail plugin, 
+  ## and will not be sent to Kubernetes or Sumo. 
+  ## This can be a comma-separated list as well. See in_tail documentation for more information. 
+  excludePath: ""
```

Comment:
We no longer use the `EXCLUDE_PATH` regex environment variable. Instead, users can update the `values.yaml` file directly under the `fluent-bit` section [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/values.yaml#L159) to change which files they wish to collect. Let's get rid of this reference